seed/challenges/basic-javascript.json

```diff
@@ -974,13 +974,13 @@
         "Create an <code>if-else statement</code> to return <code>heads</code> if the flip var is zero, or else return <code>tails</code> if it's not."
       ],
       "tests":[
-        "assert((function(){if(myFunction() === \"heads\" || myFunction() === \"tails\"){return true;}else{return false;}})(), 'myFunction should either return heads or tails');",
-        "assert(editor.getValue().match(/if/g).length >= 3, 'You should have created a new if statement');",
-        "assert(editor.getValue().match(/else/g).length >= 2, 'You should have created a new else statement');"
+        "assert((function(){var result = myFunction();if((result === \"heads\" && flip === 0) || (result === \"tails\" && flip !== 0)){return true;}else{return false;}})(), '<code>myFunction</code> should either return <code>heads</code> or <code>tails</code> if all conditions met.');",
```

Comment:
I'd recommend splitting this out to be two tests (like [here](https://github.com/FreeCodeCamp/FreeCodeCamp/pull/2158#issuecomment-138386354)):
1. one that tests that the result is either `heads` or `tails` - `myFunction should either return heads or tails`
2. one that tests if flip is the correct value - `myFunction should return heads when flip equals 0 and tails when flip equals 1`